packages/app/src/specs/generators/component/ComponentGenerator.tsx

```diff
@@ -0,0 +1,21 @@
+import { filters } from '../GeneratorsCommon'
+import ComponentGeneratorStepOne from './ComponentGeneratorStepOne.vue'
+import type { SpecGenerator } from '../types'
+import ComponentGeneratorCard from './ComponentGeneratorCard.vue'
+
+export const ComponentGenerator: SpecGenerator = {
+  card: ComponentGeneratorCard,
+  entry: ComponentGeneratorStepOne,
+  disabled: () => { },
```

Comment:
Do we ever have a disabled state? We have a lot of baggage from our previous implementation that is hanging around